src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const isWoman = person.sex === 'f';
```

Comment:
Checking for a woman using `person.sex === 'f'` is clear, but if you find yourself repeating this logic, you might want to consider creating a utility function or encapsulating the logic within the `Person` model as a method.